test/sequential/test-child-process-execsync.js

```diff
@@ -112,6 +112,8 @@ assert.strictEqual(ret, `${msg}\n`,
 // Verify the execFileSync() behavior when the child exits with a non-zero code.
 {
   const args = ['-e', 'process.exit(1)'];
+  const spawnSyncResult = spawnSync(process.execPath, args);
+  const spawnSyncKeys = Object.keys(spawnSyncResult);
```

Comment:
Suggestion:
```js
const spawnSyncKeys = Object.keys(spawnSyncResult).sort();
assert.deepStrictEqual(spawnSyncKeys, [ /* ... */ ]);
```
Right now you check that the corresponding properties exist on the error object but that won't catch regressions where the result of `spawnSync()` lacks some (or all) properties.

Reply:
done